src/cdk/layout/breakpoints-observer.ts

```diff
@@ -60,9 +60,14 @@ export class BreakpointObserver implements OnDestroy {
     const observables = queries.map(query => this._registerQuery(query).observable);
 
     return combineLatest(observables).pipe(map((breakpointStates: BreakpointState[]) => {
-      return {
-        matches: breakpointStates.some(state => state && state.matches)
-      };
+      const output: BreakpointState = {matches: false};
```

Comment:
I don't see how this is different from what it was before. Can you explain and add a comment?

This also seems like it should have a test